tools/validators/instance_validator/generate_universe.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates the ontology universe for the instance validator."""
+
+from __future__ import print_function
+
+from sys import path
+import os
+
+# add universe building packages to path
```

Comment:
what would happen if the ontology moves? can you check if there are files and exit when there is not

Reply:
Yes, I have a check for the ontology being in the right place inside of `build_universe()`.